statistics-client-ingest/src/test/java/no/difi/statistics/client/IngestClientTest.java

```diff
@@ -0,0 +1,119 @@
+package no.difi.statistics.client;
```

Comment:
Vurder å bruke [WireMock](wiremock.org) for disse testene. Du kan kanskje få litt inspirasjon [herfra](http://confluence/x/BgPTAQ).